wagtail_streamfield_migration_toolkit/management/commands/streamdatamigration.py

```diff
@@ -0,0 +1,232 @@
+from functools import lru_cache
+from django.core.management.base import BaseCommand, CommandError
+from django.db import connection
+from django.db.migrations.loader import MigrationLoader
+from django.db.migrations import Migration
+from django.db.migrations.autodetector import MigrationAutodetector
+from django.db.migrations.state import ProjectState
+from django.db.migrations.writer import MigrationWriter
+from django.apps import apps
+from wagtail.blocks import StreamBlock, StructBlock
+
+from wagtail_streamfield_migration_toolkit.migrate_operation import MigrateStreamData
+from wagtail_streamfield_migration_toolkit.operations import (
+    RemoveStreamChildrenOperation,
+    RenameStreamChildrenOperation,
+    RemoveStructChildrenOperation,
+    RenameStructChildrenOperation,
+)
+
+
+class Command(BaseCommand):
+
+    help = "Generate data migrations with the given operations. Note that you can only \
+        generate changes for models in a single app at a time."
+
+    def add_arguments(self, parser):
+        parser.add_argument("--name", help="Use this name for migration file")
+        subparsers = parser.add_subparsers(dest="operation_type")
+
+        rename_parser = subparsers.add_parser(
+            "rename", help="rename a block. rename --help to see options"
+        )
+        rename_parser.add_argument("app_label", help="app name")
+        rename_parser.add_argument("old_name", help="name of the block to be renamed")
+        rename_parser.add_argument("new_name", help="new name")
+        rename_parser.add_argument(
+            "paths",
+            nargs="+",
+            action="extend",
+            help="path/s to the block/s which is being operated on: '<model_name>.<field_name>.?<block_path>' \
```

Comment:
```suggestion
            help="path/s to the block/s which is being operated on: '<model_name>.<field_name>[.<block_path>]' \
```

Small suggestion - this might be clearer. The existing way suggests that you could have `<field_name><non_empty_block_path>` (without the delimiting `.`)